spec/Powertrain/Battery.vspec

```diff
@@ -127,7 +127,7 @@ Charging.ChargePortFlap:
 Charging.ChargePlugStatus:
   datatype: boolean
   type: sensor
-  description: Signal indicating if charge plug is connected or not.
+  description: Signal indicating if a charging cable is connected to the charge plug or not.
```

Comment:
I am not 100% sure on the intention of this signal. If the intention is to focus on the cable or on the connection to a charger? I.e. if you connect a cable to a vehicle, but do not connect the other end to a charger, shall this signal then anyway shall true? Or shall the focus rather be on if the vehicle is connected to a functional/powered charger?

Reply:
Meeting: Change name to "IsChargingCableConnected"